ignite/handlers/checkpoint.py

```diff
@@ -205,6 +205,11 @@ def __call__(self, engine: Engine) -> None:
 
             checkpoint = self._setup_checkpoint()
 
+            # happens in distributed setting when not in first process.
+            # Avoids that all processes read (not necessary due to parameter syncing)
+            if checkpoint is None:
```

Comment:
Sorry, I do not get your comments above why to do this.

Reply:
This is to exit the function for all the processes not performing the actual saving